app/assets/javascripts/controllers/middleware_deployment/middleware_deployment_controller.js

```diff
@@ -29,15 +29,14 @@ function MwAddDeploymentController($scope, $http, miqService) {
             angular.element('#modal_d_div').modal('hide');
           }
           miqService.miqFlash(result.data.status, result.data.msg);
+          miqService.sparkleOff();
         },
         function() { // error
-          var msg = sprintf(__('Unable to deploy %s on this server %s'), data.runtimeName,
+          var msg = sprintf(__('Unable to deploy %s on this server %s"'), data.runtimeName,
               (isGroupDeployment ? ' group.' : '.'));
           miqService.miqFlash('error', msg);
           angular.element('#modal_d_div').modal('hide');
-        })
```

Comment:
why should we want to revert this change?

Reply:
The idea was to revert `miq_service.js` file, and still have a working deployment controller and working specs if any.
But honestly, I didn't care much what happened with this file when reverting the involved commit. The UI is hidden and I'll place a follow-up PR removing this controller completely.

Reply:
Ok, in that case I suggest that you do the removal in one PR, please.

I don't think that it's useful to spend time reviewing changes to a file that is going to be removed.

Reply:
Right, I removed that revert and will introduce it in the follow up PR.